src/nvim/eval.c

```diff
@@ -6603,8 +6603,9 @@ void common_function(typval_T *argvars, typval_T *rettv, bool is_funcref, FunPtr
                          : (const char *)s));
     // Don't check an autoload name for existence here.
   } else if (trans_name != NULL
-             && (is_funcref ? find_func(trans_name) == NULL
-                            : !translated_function_exists((const char *)trans_name))) {
+             && (is_funcref
+                 ? find_func(trans_name) == NULL
```

Comment:
Could uncrustify be configured to force multi-line ternaries or 1-liner ternaries (if within line length)? Could be a problem with nested ternaries that use `()`.

Reply:
Maybe. I've disabled pretty much all rules on ternary operators since they're somehow the only things the style guide has nothing to say about. It's basically impossible to follow the style guide programmatically so disabling formatting on ternary operators was a cheap strategy to satisfy clint.